### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by acenasir**

### CODE REVIEW

**Comment by MadhushaPrasad** [comment]
can you please add some valuable text above the Figma link? And also export the Home page and attached it to the readme file

File: README.md [hunk 1]
```diff
@@ -87,3 +87,10 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+
+ <hr>
+
```
Comment (MadhushaPrasad) [inline README.md:92]: can you please add some valuable text above the Figma link?

File: README.md [hunk 2]
```diff
@@ -87,3 +87,10 @@ You can contribute to the project on github. Here's how you can do it.
...
+<a href="https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782" target="_blank"> Figma Designs </a>
+
+</br>
+
```
Comment (MadhushaPrasad) [inline README.md:96]: export the Home page and attached it here

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -87,3 +87,7 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+<hr>
+## 🔖 Figma Link for this project
```
Comment (MadhushaPrasad) [inline README.md]: fix this ## tags

File: README.md [hunk 4]
```diff
@@ -87,3 +87,7 @@ You can contribute to the project on github. Here's how you can do it.
...
+<a href="https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782" target="_blank"> Figma Designs </a>
+</br>
```
Comment (MadhushaPrasad) [inline README.md:94]: export the Home page from the Figma file and attached it in here

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (acenasir) [reply README.md]: Added

File: README.md (hunk 4 above)
Comment (MadhushaPrasad) [reply README.md:94]: [here is the figma link](https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782)

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (acenasir) [reply README.md]: Please give me access to the figma designs

File: README.md (hunk 4 above)
Comment (acenasir) [reply README.md:94]: Grant access

File: README.md (hunk 2 above)
Comment (MadhushaPrasad) [reply README.md]: now you can access the figma file

Comment (MadhushaPrasad) [reply README.md]: @acenasir

Comment (acenasir) [reply README.md]: Added Home page png export into root folder

File: README.md (hunk 4 above)
Comment (MadhushaPrasad) [reply README.md:94]: no need to add .png to the repository, actually I said you to attach that image to the readme

**Review by MadhushaPrasad — CHANGES_REQUESTED** [review]

**Comment by acenasir** [comment]
Added Figma Home Page screenshot into readme.md

File: README.md (hunk 2 above)
Comment (IndudiniThennakoon) [reply README.md:97]: @acenasir Please delete the Home.png and update README file using GitHub file update. You can drag the home image and drop it in to the bottom of the README file and wait until it update, then send a pull request.

Comment (IndudiniThennakoon) [reply README.md:97]: @acenasir Please delete the Home.png and update README file using GitHub file update. You can drag the home image and drop it in to the bottom of the README file and wait until it update, then send a pull request.

**Review by IndudiniThennakoon — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -87,3 +87,11 @@ You can contribute to the project on github. Here's how you can do it.
 <a href="https://www.w3.org/html/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/html-5.png"/> </a> <a href="https://www.w3schools.com/css/" target="_blank"> <img src="https://img.icons8.com/color/48/000000/css3.png"/> </a><a href="https://www.php.net/" target="_blank"> <img src="https://img.icons8.com/dusk/64/000000/php-logo.png"/>
 
 </br>
+<hr>
+<h2>🔖 Figma Designs</h2>
+</br>
+<a href="https://www.figma.com/file/dz4JS3XBiGzTDGGIQ3PczB/Diagon-Alley?node-id=20%3A782" target="_blank"> Figma Link for this project</a>
+</br>
+![image](https://user-images.githubusercontent.com/5544472/196491229-06cb8dfe-215c-481e-a3a5-56eaea008d6f.png)
```
Comment (MadhushaPrasad) [inline README.md]: fix the code. I cant see the image

**Pull request closed without merging**